proposals/collection-expression-arguments.md

```diff
@@ -398,13 +398,56 @@ Should arguments with `dynamic` type be allowed? That might require using the ru
 
 ## Open questions
 
-### Target types where arguments are *required*
+### Conversions
+
+Should collection arguments and the applicable methods affect convertibility of the collection expression?
+
+```csharp
+Print([with(comparer: null), 1, 2, 3]); // ambiguous or Print<int>(HashSet<int>)?
+
+static void Print<T>(List<T> list) { ... }
+static void Print<T>(HashSet<T> set) { ... }
+```
```

Comment:
`Print<int>(new(comparer: null))` gives:

> CS0121: The call is ambiguous between the following methods or properties: 'C.Print<T>(List<T>)' and 'C.Print<T>(HashSet<T>)'

If that's ambiguous, it would be intuitive to me for `Print<int>([with(comparer: null)])` to also be ambiguous.

Reply:
agreed.  and i imagine the same holds true of `Print<int>(new(comparer: null) { 1, 2, 3 })`.

Reply:
Thanks. I've added those examples.